pdns/dnsdistdist/dnsdist-idstate.cc

```diff
@@ -80,6 +86,15 @@ void InternalQueryState::sendDelayedProtobufMessages() const
     // that only contains the OTTrace data as a single bytes field
     msg_logger.second->queueData(msg_logger.first + otPBBuf);
   }
+
+  static thread_local string minimalPBBuf;
```

Comment:
Could we reuse `otPBBuf`? `thread_local` is not cheap, otherwise I wouldn't bother.